irohad/ametsuchi/impl/storage_impl.cpp

```diff
@@ -69,11 +77,11 @@ namespace iroha {
           "SET SESSION CHARACTERISTICS AS TRANSACTION READ ONLY;");
     }
 
-    std::unique_ptr<TemporaryWsv> StorageImpl::createTemporaryWsv() {
+    expected::Result<std::unique_ptr<TemporaryWsv>, std::string> StorageImpl::createTemporaryWsv() {
```

Comment:
`using iroha::expected::Result` but you use `expected::Result`. IMO it can be either Result, or you can remove using.

UPD: also I've noticed the same across all the file.

If you want to use `expected::Result` and others, you can do it without `using`.